test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/MvcTagHelpersWebSite.Customer.Index.html

```diff
@@ -0,0 +1,42 @@
+﻿
```

Comment:
I thought the convention is test web site name + controller name + action name??